include/linux/kern_levels.h

```diff
@@ -23,6 +23,8 @@
  */
 #define KERN_CONT	KERN_SOH "c"
 
+#define KERN_TIME	KERN_SOH "t"	/* already prefix with the timestamp */
```

Comment:
Neither the commit message nor the definition here are clear. What embedded processor are you talking about and is the timestamp equivalent (could a a different counter or clock source?).

also please fix typo, it's ugly not uglily.

Reply:
The embedded processor mean DSP/MCU in SoC,
Yes, the time base on DSP/MCU may different from kernel, but the unit and format is same as kernel.